pennylane/beta/plugins/default_tensor_tf.py

```diff
@@ -227,7 +227,7 @@ class DefaultTensorTF(DefaultTensor):
     # pylint: disable=too-many-instance-attributes
     name = "PennyLane TensorNetwork (TensorFlow) simulator plugin"
     short_name = "default.tensor.tf"
-    _capabilities = {"model": "qubit", "tensor_observables": True, "provides_jacobian": True}
+    _capabilities = {"model": "qubit", "tensor_observables": True, "provides_jacobian": True, "passthru": True}
```

Comment:
Question: do we want to make the capability key/value something like the following?
```python
{"classical_backprop": "tf"}
```
Allowed keys could be `"autograd"`, `"torch"`, `None` or `False`.

Not quite sure what the use case is, but it provides a bit more information for the future. What do you think? Any other suggestions? I'm not 100% sold on this suggestion myself!

Reply:
Right! This could be a really good idea if there is a specific case in which the extra information about the framework could be useful. If right now there would be no case, then maybe it's better to keep the logic as simple as we can, but also keep in mind that this option could help us out in the future. :)

Reply:
@antalszava: I added `{"passthru_interface": "tf"}` as a capability, as I noticed a user might do

```python
@qml.qnode(dev, diff_method="classical", interface="torch")
```
which will fail if the device does not support PyTorch for classical backprop (e.g., `default.tensor.tf`).

Reply:
Cool! How about having this option only (and discarding `"passthru"`) in `capabilities`? The existence of `passthru_interface` should suffice, right?

Reply:
Sounds good, that makes sense!

Reply:
Your change looks 💯